src/server/index.js

```diff
@@ -11,6 +11,7 @@ const generateToken = require('./api/login/generateToken.js');
 const jwtVerify = require('./jwtVerify.js');
 const register = require('./api/register/register.js');
 const boards = require('./api/boards/boards.js');
+const jsonParser = bodyParser.json();
```

Comment:
we don't need this since we set the `bodyParser.json()` as a middleware at line 29 (`app.use(bodyParser.json())`)